vault.js

```diff
@@ -1,4 +1,35 @@
+/*jshint esversion: 6 */
 'use strict';
-module.exports = function() {
-  
-};
\ No newline at end of file
+module.exports = function(){
+
+var vault = {};
+
+  function setValue(key, value){
+     vault[key] = value;
+  }
+
+  function getValue(key){
```

Comment:
This code can be simplified. You should only need to check key once.

Also, there is an edge case you need to watch for 
```
var vault = VaultGenerator();

console.log( vault.getValue("toString")); // [Function: toString]
// should be null;
```